project_classification/project_classification_view.xml

```diff
@@ -46,10 +48,10 @@
       <field name="model">project.project</field>
       <field name="inherit_id" ref="project.edit_project" />
       <field name="arch" type="xml">
-        <xpath expr="/form/sheet/notebook/page[@string='Other Info']/group/group[3]/field[@name='parent_id']" position="replace">
-          <field name="classification_id" widget="selection" on_change="onchange_classification_id(classification_id)"/>
+        <xpath expr="/form/sheet/notebook/page[@string='Other Info']//field[@name='parent_id']" position="replace">
+          <field name="classification_id" required="1" widget="selection" on_change="onchange_classification_id(classification_id)"/>
```

Comment:
@mdietrichc2c I see an other options, ensure existing project gets a classification_id while an init script like we did for some other modules.

Here is an exemple using a pre_init_hook to fill a required field with a default value for old records:

https://github.com/OCA/stock-logistics-workflow/blob/8.0/stock_ownership_availability_rules/__openerp__.py#L32
https://github.com/OCA/stock-logistics-workflow/blob/8.0/stock_ownership_availability_rules/__init__.py#L4-L9